api/testing-internal/src/main/java/io/opentelemetry/api/testing/internal/AbstractOpenTelemetryTest.java

```diff
@@ -115,4 +115,11 @@ void toString_noop_Valid() {
                 + "propagators=DefaultContextPropagators{textMapPropagator=NoopTextMapPropagator}"
                 + "}");
   }
+
+  @Test
+  void neverReturnsObfuscatedNoop() {
+    assertThat(GlobalOpenTelemetry.get()).isSameAs(OpenTelemetry.noop());
+    // ensure sequential calls of GlobalOpenTelemetry.get() return same object
+    assertThat(GlobalOpenTelemetry.get()).isSameAs(OpenTelemetry.noop());
```

Comment:
I think that storing the result of `OpenTelemetry.noop()` as a local variable and using that for both comparisons would be a small improvement on this test.